lib/measured/measurable.rb

```diff
@@ -1,12 +1,13 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
-  attr_reader :unit, :value
+  attr_reader :unit, :value, :unit_system
 
-  def initialize(value, unit)
+  def initialize(value, unit, unit_system)
```

Comment:
The unfortunate side-effect of these changes, as they stand, is having to pass in a unit system. I could technically avoid this if I dynamically define a subclass of `Measurable` that has a class method that points to the `UnitSystem` that `Measured.build` builds.

What I'd really love to be able to do is have the units reference the system they're in, but this would result in a circular dependency, which I'm not a fan of (and would perhaps require mutation). Another option would be to have all of the methods in `UnitSystem` that can return a `Unit` instance return a wrapped `Unit` (e.g., `UnitWithSystem`), but then we'd impose the performance penalty of constructing those instances all of the time.